bootstrap.sh

```diff
@@ -66,6 +74,18 @@ if [[ $(uname) == "Darwin" ]]; then
   GROUP_ID=200
 fi
 
+
+# Test for tag override
+if [ -n "${TAG_OVERRIDE}" ]; then
+    # Currently, tag override conflicts with release.
+    if [ -n "${RELEASE_VERSION}" ]; then
+        echo "--tag conflicts with --release, exiting"
+        exit 1
+    fi
+    docker build --no-cache --build-arg USER_ID=0 --build-arg GROUP_ID="$GROUP_ID" $LEAP_ARGUMENT $CDT_ARGUMENT -f Dockerfile.unix -t "${TAG_OVERRIDE}" "$SDIR"
```

Comment:
Can you please extract this `docker build` to a function and use this function in lines 85 and 89?

Reply:
Fixed in 38b74ae. Also added more comments and attempted to make everything a bit clearer.